res/css/focus.css

```diff
@@ -0,0 +1,13 @@
+/* the box shadow is taken from photon-button primary button focus styling
+https://design.firefox.com/photon/components/buttons.html */
+
+/* removes the default Firefox focus indicator,
+so that we can add a custom indicator */
+input::-moz-focus-inner, button::-moz-focus-inner {
+  border: 0;
+}
```

Comment:
This isn't enough for the "retrieve profiles" button in https://deploy-preview-1674--perf-html.netlify.com/compare/ (at least on Linux), but maybe it's to be expected because that button should really be a photon-button. I think the problem is that that button is still a "widget" button instead of a "CSS-driven button". Let's not focus on this in this PR, I think we should just convert that button to a photon button in a separate PR.